jaxws-ri/rt/src/main/java/com/sun/xml/ws/util/xml/XMLStreamWriterFilter.java

```diff
@@ -53,6 +53,26 @@ public void writeStartDocument() throws XMLStreamException {
         writer.writeStartDocument();
     }
 
+    private boolean isUnusualChar(char c){
+        // all char points between 0 and 31 excluding the allowed white spaces (9=\t, 10=\n, 13=\r)
```

Comment:
You missed my point. I meant to remove this useless comment and use \t... in code.